src/sentry/models/orgauthtoken.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+from django.utils.encoding import force_text
+
+from sentry.db.models import (
+    ArrayField,
+    BaseManager,
+    FlexibleForeignKey,
+    Model,
+    control_silo_only_model,
+    sane_repr,
+)
+from sentry.models.orgauthtokenproject import OrgAuthTokenProject
+
+
+@control_silo_only_model
+class OrgAuthToken(Model):
+    __include_in_export__ = True
+
+    organization = FlexibleForeignKey("sentry.Organization", null=False)
+    projects = models.ManyToManyField(
+        "sentry.Project", related_name="projects", through=OrgAuthTokenProject
+    )
+    # max JWT token length is 2048, +7 for our custom prefix (sntrys_)
+    token = models.CharField(max_length=2055, unique=True, null=False)
+    label = models.CharField(max_length=255, null=False)
+    scope_list = ArrayField(of=models.TextField)
+
+    user_added = FlexibleForeignKey("sentry.User", null=True)
+    date_added = models.DateTimeField(default=timezone.now, null=False)
+    date_last_used = models.DateTimeField(null=True)
```

Comment:
Instead of having a `null` field, we could have it default to `timezone.now` and update it when the user uses it, but this changes the semantics. I don't know which ones are more representative of the situation.

Reply:
IMHO it's useful to know that the token was never used, I'd say?

Reply:
You could technically infer it if the two dates are equal but ofc it's not the best way. My main concern was that having nullability in the db is not very nice but I understand your point.